content/en/docs/concepts/services-networking/service.md

```diff
@@ -184,73 +189,94 @@ spec:
       targetPort: 9376
 ```
 
-Because this Service has no selector, the corresponding Endpoints object is not
-created automatically. You can manually map the Service to the network address and port
-where it's running, by adding an Endpoints object manually:
+Because this Service has no selector, the corresponding EndpointSlice (and
+legacy Endpoints) objects are not created automatically. You can manually map the Service
+to the network address and port where it's running, by adding an EndpointSlice
+object manually. For example:
 
 ```yaml
-apiVersion: v1
-kind: Endpoints
+apiVersion: discovery.k8s.io/v1
+kind: EndpointSlice
 metadata:
-  # the name here should match the name of the Service
-  name: my-service
-subsets:
+  name: my-service-1 # by convention, use the name of the Service
+                     # as a prefix for the name of the EndpointSlice
+  labels:
```

Comment:
should we suggest to add the other label that indicates the owner?
```
	LabelManagedBy = "endpointslice.kubernetes.io/managed-by"
```

Reply:
:thinking: What would we set the value of that label to? The name of the user who is manually adding the EndpointSlice?

Reply:
I'm minded to suggest that documenting `endpointslice.kubernetes.io/managed-by` here belongs as a separate change in its own PR.